ProjectCoimbra.UWP/Project.Coimbra/Pages/GamePage.xaml.cs

```diff
@@ -68,6 +71,10 @@ public GamePage()
                     (pitchNoteName) => string.Equals(pitchNoteName, noteName, StringComparison.Ordinal)))?.Index;
         }
 
+#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
+        private void MidiEngine_PlaybackFinished(object sender, EventArgs e) => CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () => this.Frame.Navigate(typeof(ModePage), null, new DrillInNavigationTransitionInfo()));
```

Comment:
Part of this can be moved to the `using` statements.

Reply:
Assuming you are referring to CoreApplication.*, it's actually a static class with properties.

Reply:
I was referring to `CoreApplication`. Thanks for clarifying.